src/react-apps/applications/service-development/src/config/redirects.tsx

```diff
@@ -0,0 +1,10 @@
+export const redirects = ([
+  {
+    from: '/',
+    to: '/aboutservice',
+  },
+  {
+    from: '/create',
+    to: '/uieditor',
+  },
+]);
```

Comment:
Could you tell me the purpose of this file?

Reply:
Separating all redirects in Service-Development in its own "configuration".

The selected sub header menu will redirect to a specific SubApp, which will eventually change during the development face. The subheader menus will not have "its own page".